main.py

```
@@ -0,0 +1,37 @@
+import ai_adapter
+import asyncio
+import ai_adapter
```

Ruff (0.8.0)

Redefinition of unused `ai_adapter` from line 1

Remove definition: `ai_adapter`

(F811)